ig-data/input/pagecontent/index.md

```diff
@@ -9,30 +9,30 @@
 
 <h1>ICAREdata and mCODE</h1>
 
-<p>This implementation guide intends to cover content necessary for ICAREdata that is not part of the <a href="http://standardhealthrecord.org/guides/mcode/" target="_blank">mCODE 0.9.1 DSTU2 specification</a>. Implementers should refer directly to mCODE for the specification of all mCODE elements.</p>
+<p>This implementation guide intends to cover content necessary for ICAREdata that is not part of the <a href="http://standardhealthrecord.org/guides/mcode/" target="_blank">mCODE v1.0 R4 specification</a>. Implementers should refer directly to mCODE for the specification of all mCODE elements.</p>
 <p>The mCODE implementation guide includes information on how to interpret conformance requirements as well as elements marked as "must support". This implementation guide follows those same guidelines.</p>
 
-<h1>Structured phrases, mCODE and the ICAREData IG</h1>
+<h1>Structured phrases, mCODE and the ICAREdata IG</h1>
 
-<p>The ICAREdata&reg strategy is to collect improved real-world data in a way does not disrupt existing clinical processes and minimizes burden to the provider while enabling the collection of complete, accurate, and computable data. ICAREdata questions are answered by clinicians with the results appearing as structured phrases in the clinical note.</p>
+<p>The ICAREdata strategy is to collect improved real-world data in a way that does not disrupt existing clinical processes and minimizes burden to the provider while enabling the collection of complete, accurate, and computable data. ICAREdata questions are answered by clinicians with the results appearing as structured phrases in the clinical note.</p>
 
-<p>ICAREdata structured phrases questions and value sets are intended to be mapped to FHIR resources compliant with the requirements specified in the mCODE and ICAREdata implementation guides. This includes mapping 1) the ICAREdata question to the appropriate FHIR resource and element, and 2) the associated value to the appropriate vocabulary defined in the specification.</p>
+<p>ICAREdata structured phrase questions and value sets are intended to be mapped to FHIR resources compliant with the requirements specified in the mCODE and ICAREdata implementation guides. This includes mapping 1) the ICAREdata question to the appropriate FHIR resource and element, and 2) the associated value to the appropriate vocabulary defined in the specification.</p>
 
 <p>The figure below illustrates how ICAREdata structured phrases relate to FHIR profiles and implementation guides.</p>
-<img src="icaredata_fhir.png" alt="ICAREdata and FHIR" width="65%" align="middle"/>
+<img src="icaredata_fhir.png" alt="ICAREdata and FHIR" width="65%" align="middle" style="margin: 0px 400px 0px 0px;"/>
```

Comment:
the _align_ attribute is not supported in HTML5. To center the graphic, you have to use CSS.

Reply:
Right now they don’t have an enduring email address, so for now it’ll just have to be Ammu’s